query2.php

```diff
@@ -68,9 +66,9 @@
 								}
 							?>
 			</div>
-
+			</center>
 			<!-- This class is for my main content-->	
-			<div style="height: 100%;" class="content"> 
```

Comment:
#made the div fit all content - redundant in second version as there was a scroll area used